DotnetViewComponents/ViewModels/TableViewModel.cs

```diff
@@ -0,0 +1,45 @@
+namespace DotnetViewComponents.ViewModels
+{
+    public class TableViewModel
+    {
+        public TableViewModel(
+            string title,
+            List<string> headers,
+            List<List<TableColumnModel>> rows,
+            bool isResponsive = false,
+            bool hasNumericData = false)
+        {
+            Title = title;
+            Headers = headers;
+            Rows = rows;
+            IsResponsive = isResponsive;
+            HasNumericData = hasNumericData;
+        }
+
+        public string Title { get; set; }
+
+        public List<string> Headers { get; set; }
```

Comment:
Could we simplify this by taking the headers from the first entry in Rows ? This would reduce chances of mismatched aria-labels too.